src/demo/containers/playground/Playground.tsx

```diff
@@ -1,104 +1,81 @@
 import * as React from 'react'
-import { Link, Route } from 'react-router-dom'
+import { Link, Route, Switch } from 'react-router-dom'
 import CardContainerLogicDemo from './CardContainerLogicDemo'
 import ModalDownloadDemo from './ModalDownloadDemo'
-import ShowDownloadDemo from './ShowDownloadDemo'
 import FormServicesIntegrationDemo from './FormServicesIntegrationDemo'
 import DownloadListTable from '../../../lib/containers/download_list/DownloadListTable'
 import { WidgetDemo } from './WidgetDemo'
-import { RouteChildrenProps } from 'react-router'
 import { AccessRequirementDemo } from './AccessRequirementDemo'
 import TemplateComponentDemo from './TemplateComponentDemo'
 import Resources from '../../../lib/containers/home_page/resources/Resources'
 import { SynapsePlotDemo } from './SynapsePlotDemo'
 import { ExternalFileHandleLink } from '../../../lib/containers/ExternalFileHandleLink'
 import ColorPaletteInspector from './ColorPaletteInspector'
+import { useRouteMatch } from 'react-router'
 
 /**
  * Demo of features that can be used from src/demo/utils/SynapseClient
  * module
  */
-const App = ({
-  match,
-}: {
-  match?: RouteChildrenProps['match']
-  token: string
-}) => {
-  if (!match) {
-    return <div />
-  }
+const App = () => {
+  const { url } = useRouteMatch()
```

Comment:
Many of the React Router v5 -> v6 migration steps can be summarized as "use hooks instead of passing props, when possible".